AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+e-tesettur.com.tr###cookie
```

Comment:
please add the domain to the existing rule

Reply:
I'm sorry but which one?

Reply:
I'll show you.